src/x509/clu_x509_sign.c

```diff
@@ -553,13 +678,21 @@ WOLFCLU_CERT_SIGN* wolfCLU_readSignConfig(char* config, char* sect)
         }
     }
 
+    /* does the subject need to be unique? */
+    if (ret != NULL) {
+        tmp = wolfSSL_NCONF_get_string(conf, CAsection, "unique_subject");
+        if (tmp != NULL && XSTRCMP(tmp, "yes") == 0) {
```

Comment:
Consider using can insensitive version? What if config has "yes123" this would still pass, is that okay?